doc/usage.rst

```diff
@@ -458,10 +459,15 @@ use :meth:`pooch.Pooch.load_registry` to read them:
         # We'll load it from a file later
         registry=None,
     )
-    GOODBOY.load_registry(os.path.join(os.path.dirname(__file__), "registry.txt"))
-
-The ``registry.txt`` file in this case is in the same directory as the ``datasets.py``
-module and should be shipped with the package. It's contents are:
+    # Get registry file from package_data
+    registry_file = pkg_resources.resource_stream("plumbus", "registry.txt")
+    # Load this registry file
+    GOODBOY.load_registry(registry_file)
+
+The ``registry.txt`` file in this case is in the ``plumbus/`` directory module and should be
+shipped with the package. We use `pkg_resources <https://setuptools.readthedocs.io/en/latest/pkg_resources.html#basic-resource-access)`__
```

Comment:
Maybe instead of "We use package_resources" you could say: 

> We access the file contents using pkg_resources (instead of `os.path.dirname(__file__)`) because it's safer alternative. For example, it makes sure our code can work out of a zip archive, which is necessary depending on how the user installs the package.